css/services.css

```diff
@@ -0,0 +1,130 @@
+/* SERVICES PAGE */
+/* Services Header */
+.services-header {
+  display: flex;
+  flex-direction: column;
+  border-bottom: 1px solid black;
+}
+
+.services-header .services-header-img {
+  flex: 1;
+}
```

Comment:
I would set this to width: 100% instead of flex:1, this means the image keeps its correct proportions